active/0000-daemonize.md

```diff
@@ -0,0 +1,76 @@
+- Start Date: 2015-05-26
+- RFC PR: (leave this empty)
+- ember-cli Issue: (leave this empty)
+
+# Summary
+
+ember-cli will run as a daemon in the background, rather than being started for each `ember` command.
+
+# Motivation
+
+Currently running an `ember` command can take quite some time, especially when a lot of addons are installed.
+This is mainly due to the "boot up" period, when Node.js / io.js loads ember-cli itself and *all* the addons.
```

Comment:
I don't actually think this is correct as `ember-cli` legitimately starts in a few hundred milliseconds. All the commands that take time, are because they are doing something expensive, like building/rebuilding.

If performance on those commands is an issue (which I believe it is) they should be improved directly.

My work on zeus (the fast-boot for rails) https://github.com/burke/zeus, was to deal with the extremely slow `require` times in large add-ons.  Currently ember-cli does not demonstrate this same issues.